Specs/Number/French/NumberRangeModel

```diff
@@ -0,0 +1,1359 @@
+          "value": "[5000,)"
+        },
+        "Start": 14,
+        "End": 33
+      }
+    ]
+  },
+  {
+    "Input": "Son score est égal à 5000 ou plus de 4500",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "égal à 5000",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[5000,5000]"
+        },
+        "Start": 14,
+        "End": 25
+      },
+      {
+        "Text": "plus de 4500",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(4500,)"
+        },
+        "Start": 29,
+        "End": 41
+      }
+    ]
+  },
+  {
+    "Input": "Son score est inférieur ou égal à 5000",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "inférieur ou égal à 5000",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,5000]"
+        },
+        "Start": 14,
+        "End": 38
+      }
+    ]
+  },
+  {
+    "Input": "Son score est supérieur ou égal à 5000",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "supérieur ou égal à 5000",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[5000,)"
+        },
+        "Start": 14,
+        "End": 38
+      }
+    ]
+  },
+  {
+    "Input": "Son score est plus de ou égal à 5000",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "plus de ou égal à 5000",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[5000,)"
+        },
+        "Start": 14,
+        "End": 36
+      }
+    ]
+  },
+  {
+    "Input": "Son score est plus de 5000 ou égal à 6000",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "plus de 5000",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(5000,)"
+        },
+        "Start": 14,
+        "End": 26
+      },
+      {
+        "Text": "égal à 6000",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[6000,6000]"
+        },
+        "Start": 30,
+        "End": 41
+      }
+    ]
+  },
+  {
+    "Input": "Son score est égal à 5000 ou moins de 5000",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "égal à 5000",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[5000,5000]"
+        },
+        "Start": 14,
+        "End": 25
+      },
+      {
+        "Text": "moins de 5000",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,5000)"
+        },
+        "Start": 29,
+        "End": 42
+      }
+    ]
+  },
+  {
+    "Input": "La gamme de nombres est de 1000 à 5000",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "1000 à 5000",
```

Comment:
"de 1000 à 5000"

Reply:
According to the rules, French translation should be consistent with original English data and we should try to avoid preposition. Therefore, for example, if an English output text is "from 1000 to 5000'', the translation will be "de 1000 à 5000", but if the English one is "1000 to 5000'' or "1000-5000'', the French output will also be "1000 à 5000".